cmd/up/up.go

```diff
@@ -963,6 +970,9 @@ func (up *upContext) shutdownHybridMode() {
 		return
 	}
 
+	if err := up.hybridCommand.Cancel(); err != nil {
```

Comment:
This is equal to running the code below of this, but this one doesn't leave the command zombie. We could remove the other command cleanup code later. WDYT?

Reply:
@jLopezbarb do you mean using `up.hybridCommand.Cancel()` instead of `terminateProcess(up.hybridCommand.Process.Pid)`? I think it makes sense, but I would test it a few times to make sure.

In regards to killing child processes, I think we still need it